doc/api/http.md

```diff
@@ -1977,6 +1977,8 @@ changes:
   * `family` {number} IP address family to use when resolving `host` or
     `hostname`. Valid values are `4` or `6`. When unspecified, both IP v4 and
     v6 will be used.
+  * `defaultPort` {number} Default port for the protocol. **Default:**
+    `agent.defaultPort` if an `Agent` is used, else `undefined`.
```

Comment:
If I'm understanding correctly, the next line (`port`) needs to be updated as well as this will be the default port rather than port 80 if it is set.

Reply:
Churn-heavy nit that can be ignored: The ordering of the options here seems kinda sub-optimal. I wonder if it makes sense to alphabetize them. (Maybe do it in a separate commit maybe so that it can be discarded for backports rather than causing merge conflicts?)